doc/storing-data.md

```diff
@@ -50,11 +61,20 @@ multiple_choice :question_2? do
     'q2-precalculated-answer'
   end
 
+  on_response do |response|
+    # response             => 'q2_option'
+    # responses            => ['q1_option']
+    # q1_calculated_answer => 'q1-calculated-answer'
+
```

Comment:
I've addressed this in a fixup commit: 913368d7f7b2be84f19e88d7b66225c8e643ffaf.